gems/smithy/lib/smithy/forge/client.rb

```diff
@@ -24,6 +24,7 @@ def forge
 
       private
 
+      # rubocop:disable Metrics/AbcSize
```

Comment:
I added the methods here to solve this but I guess it was just deferred. Is there a way to solve this one? Maybe if enumerator is passed to a method and that method does the yielding? Otherwise I guess it's fine to just ignore this one as it's a giant list.

Reply:
Yeah. In that case the multimethods are kinda unnecessary.